ui/app/store/actions.js

```diff
@@ -352,9 +351,15 @@ var actions = {
   createSpeedUpTransaction,
   createRetryTransaction,
 
-  approveProviderRequestByOrigin,
-  rejectProviderRequestByOrigin,
-  clearApprovedOrigins,
+  // Permissions
+  approvePermissionsRequest,
+  clearPermissions,
+  clearPermissionsHistory,
```

Comment:
This appears to be unused

Reply:
As is `clearPermissionsLog`, `selectApprovedAccount`, `legacyExposeAccounts`

Reply:
ab0d66fa1